logger/csv.py

```diff
@@ -189,32 +189,32 @@ def add_data_didi(self, data, plt, cycle):
         with open(file_path, 'a') as f:
             # f.write("".join(data_list))
             f.write("".join(test_list))
-        print('  Datei:   Daten in {}'.format(file_path))
+        print('  File:   Data in {}'.format(file_path))
 
-        # Plotfile für gnuplot yyyy-mm-dd.plt erzeugen
+        # Generate plotfile for gnuplot yyyy-mm-dd.plt
         if plt:
             file_path = self.dir + "/" + date_string +  ".plt"
             try:
                 f = open(file_path, 'r')
                 f.close()
             except OSError:
                 f = open(file_path, 'w')
-                # f.write('# Plotfile  2020-02-08.plt am 08.02.2020 um 00:00:00 \n')
-                text = '# Plotfile  ' + file_path + ' am ' + self.get_date() + ' ab ' + self.get_time() + ' \n'
+                # f.write('# Plotfile  2020-02-08.plt on 08.02.2020 at 00:00:00 \n')
+                text = '# Plotfile  ' + file_path + ' on ' + self.get_date() + ' at ' + self.get_time() + ' \n'
                 f.write(text)
                 text = 'DATAFILE = "' + self.get_date() + '.csv" \n'
                 f.write(text)
-                text = 'TITLE    = "Messdaten der BOB-Bienenbeute' + ' am ' + self.get_date() + ' ab ' + self.get_time() + '" \n'
+                text = 'TITLE    = "measured data of BOB-Bienenbeute' + ' on ' + self.get_date() + ' at ' + self.get_time() + '" \n'
```

Comment:
What's the right translation of BOB-Bienenbeute in this context? Is it just BOB-hive?